src/components/App/userReducer.ts

```diff
@@ -45,12 +52,19 @@ export const userReducer = produce(
             case SIGNOUT:
                 draft.user = {
```

Comment:
return the initState on signout is usually the simplest way

Reply:
not part of this PR